src/result.cpp

```
@@ -1,15 +1,110 @@
+}
+
+const RelevantState* get_invariant_filter(std::ostream& os) {
+    return static_cast<const RelevantState*>(os.pword(invariant_filter_index()));
+}
+
+std::ostream& operator<<(std::ostream& os, const invariant_filter& filter) {
+    os.pword(invariant_filter_index()) = const_cast<void*>(static_cast<const void*>(filter.state));
+    return os;
+}
+
+bool RelevantState::is_relevant_constraint(const std::string& constraint) const {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)